### ISSUE TITLE
The train error does not reduce

### ISSUE THREAD

**Issue by msehabibur** [microsoft/mattergen#132]
Hi team,

I trained the model on my custom dataset consisting of 540 DFT-computed structures, which includes energy, forces, and stress labels. The training completed successfully with max_epochs=500.

However, I’ve noticed that:

The total training loss stabilized around 1.35

But the position loss remains significantly higher (around 4.3–4.8) compared to the other components:

cell_train_epoch ~ 0.8

atomic_numbers_train_epoch ~ 0.11

Is this level of position loss expected for a dataset of this size and type (540 structures)?
Would you recommend any specific tuning or data normalization strategies to improve position prediction?

Thanks for your help!

[slurm-8353100.txt](https://github.com/user-attachments/files/19668559/slurm-8353100.txt)

**Comment by msehabibur** [comment]
@danielzuegner any feedback? Thank you!

**Comment by msehabibur** [comment]
[metrics.csv](https://github.com/user-attachments/files/19676641/metrics.csv)

**Comment by msehabibur** [comment]
Hi @danielzuegner,

Thank you for pointing out this issue.

I’d like to clarify that my DFT data is based on a different functional (HSE), which differs from the functional used to train the base model (primarily GGA or GGA+U). Because of this inconsistency, fine-tuning the base model wasn’t suitable for my case, and I had to train a model from scratch.

However, when I used my trained model to generate crystal structures, the results did not seem physically meaningful — I’ve attached an example for reference.

Have you considered ways to address the data scarcity problem in such cases? Specifically, is there a possibility to develop a multi-fidelity model that can incorporate data from different functionals, like combining GGA and HSE? I'd be very interested to hear your thoughts on this.

Best regards,
Habibur



data_image0
_chemical_formula_structural       SnS2GeS3AgS2
_chemical_formula_sum              "Sn1 S7 Ge1 Ag1"
_cell_length_a       381.0082678486048
_cell_length_b       91.7284626337597
_cell_length_c       33.28964614868164
_cell_angle_alpha    178.15114090950414
_cell_angle_beta     2.1117394613204192
_cell_angle_gamma    179.51257622933008

_space_group_name_H-M_alt    "P 1"
_space_group_IT_number       1

loop_
  _space_group_symop_operation_xyz
  'x, y, z'

loop_
  _atom_site_type_symbol
  _atom_site_label
  _atom_site_symmetry_multiplicity
  _atom_site_fract_x
  _atom_site_fract_y
  _atom_site_fract_z
  _atom_site_occupancy
  Sn  Sn1       1.0  0.3048764765262604  0.39237266778945923  0.7543823719024659  1.0000
  S   S1        1.0  0.8168020248413085  0.39822706580162054  0.032350525259972805  1.0000
  S   S2        1.0  0.7711313366889954  0.7489712834358215  0.8694860339164738  1.0000
  Ge  Ge1       1.0  0.6262648105621337  0.3207510709762573  0.4908522069454202  1.0000
  S   S3        1.0  0.848654270172119  0.8120518326759338  0.702378690242769  1.0000
  S   S4        1.0  0.36375054717063904  0.5166881680488586  0.9302494525909425  1.0000
  S   S5        1.0  0.3080026507377624  0.09287106245756149  0.10847681760788012  1.0000
  Ag  Ag1       1.0  0.48567730188369745  0.4951862692832947  0.03663992136716952  1.0000
  S   S6        1.0  0.98027503490448  0.016902633011341095  0.4223217666149137  1.0000
  S   S7        1.0  0.2402786016464234  0.8697946071624757  0.45834362506866455  1.0000

**Comment by danielzuegner** [comment]
Hi @msehabibur,

data sparsity is one of the use cases of fine-tuning as we describe in the paper. Others include training an ML property predictor to predict labels of additional structures. It's a hard problem because ML shines most when we have lots of data.

**Comment by ClaudioZeni** [comment]
Closing this for the moment since it has been inactive for a week.
